library/ic-ledger-types/src/lib.rs

```diff
@@ -218,6 +218,11 @@ impl AccountIdentifier {
         hex::encode(self.0)
     }
 
+    /// Provide the account identifier as bytes.
+    pub fn as_bytes(&self) -> &[u8; 32] {
+        &self.0
+    }
```

Comment:
Note: The constructor takes great care to make sure that the checksum is valid, so this method is provided, rather than making the inner value public and compromising the integrity checks.

A fixed length array is returned rather than a slice, as otherwise the caller has no guarantee that the length is correct.